woocommerce-services.php

```diff
@@ -360,10 +360,12 @@ public function load_textdomain() {
 		 */
 		public function init() {
 			$this->load_dependencies();
+
+			add_action( 'admin_init', array( $this, 'load_admin_dependencies' ) );
```

Comment:
For completeness, let's move this to `attach_hooks()`, and move the instantiation of `WC_Connect_Help_View` to `load_admin_dependencies()` so we aren't adding the WCS tab to the status page, etc, before TOS acceptance.